### ISSUE TITLE
Issue with rendering complex PDFs in Designer component on Vercel

### ISSUE THREAD

**Issue by DevRSC** [pdfme/pdfme#180]
### Describe the bug

When uploading certain PDFs in the Designer component, the PDF content is not displayed properly on the live server (deployed on Vercel). It appears as blank in the interface, but the same PDF is displayed properly on localhost. When the PDF is flattened prior to uploading, it displays correctly in all environments.

### To Reproduce

1. Fork `pdfme/playground`
2. Deploy on Vercel
3. Upload a complex PDF in the Designer component
4. PDF content appears blank

### Expected behavior

All PDFs, regardless of complexity, should be rendered correctly in the Designer component on both localhost and the live server.

### Your Environment

```markdown
Local Environment
The PDFs are correctly displayed on the local development environment.

Live Environment
The issue arises when the project is deployed on Vercel. Complex PDFs appear blank in the Designer component.
```

### Your Error Log

```shell
There is no error appearing on the live server.
```

### Additional context

ScreenShots:

Deployed Version (Vercel):
![live](https://github.com/pdfme/pdfme/assets/108634205/be07c319-22a4-41ce-ac9f-ac7107eacbe8)

Local Server:
![localhost](https://github.com/pdfme/pdfme/assets/108634205/b93ca92f-0ca1-48ad-b4d8-7b20011c1c3c)

**Comment by hand-dot** [comment]
Hi @DevRSC 

I'll check this problem.

**Comment by hand-dot** [comment]
Hi @DevRSC

Fixed pdfme version 1.2.3.
Thank you.
